dev/crypto.md

```diff
@@ -95,6 +95,38 @@ random seed.
 Algorand uses the [ed25519][ed25519] digital signature scheme to sign
 data.
 
+Algorand changes the ed25519 verification algorithm in the following way:
+- Reject if R or A (PK) are equal to one of the following: (non-canonical encoding)
+  - ```0100000000000000000000000000000000000000000000000000000000000080```
+  - ```ECFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF```
+  - ```EEFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF7F```
+  - ```EEFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF```
+  - ```EDFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF```
+  - ```EDFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF7F```
+
+- Reject if A (PK) is equal to one of the following: (small order points)
+  - ```0100000000000000000000000000000000000000000000000000000000000000```
+  - ```ECFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF7F```
+  - ```0000000000000000000000000000000000000000000000000000000000000080```
+  - ```0000000000000000000000000000000000000000000000000000000000000000```
+  - ```0000000000000000000000000000000000000000000000000000000000000000```
+  - ```C7176A703D4DD84FBA3C0B760D10670F2A2053FA2C39CCC64EC7FD7792AC037A```
+  - ```C7176A703D4DD84FBA3C0B760D10670F2A2053FA2C39CCC64EC7FD7792AC03FA```
+  - ```26E8958FC2B227B045C3F489F2EF98F0D5DFAC05D3C63339B13802886D53FC05```
+  - ```26E8958FC2B227B045C3F489F2EF98F0D5DFAC05D3C63339B13802886D53FC85```
+  - a point which holds ```(x,y) where: 2**255 -19 <= y <= 2**255```
```

Comment:
A point like mentioned is actually "non-canonical", not necessarily low-order.
See Table 2 of https://eprint.iacr.org/2020/1244.pdf
Please move to the non-canonical encoding part.

Also, can you add:

"where we remind that `y` is defined as the encoding of the point with the right-most bit cleared"